Description: Remove setting not applicable for Debian.
 Test programs abort and some tests fail because no zones are installed
 at build time when the testsuite is being run.
Author: Yavor Doganov <yavor@gnu.org>
Forwarded: not-needed
Last-Update: 2025-02-13
---

--- gnustep-base.orig/Tests/base/NSCalendarDate/TestInfo
+++ gnustep-base/Tests/base/NSCalendarDate/TestInfo
@@ -1,2 +1 @@
 # On unix systems force testing using our inbuilt time zone data.
-export GNUSTEP_BUILTIN_TZ=1
--- gnustep-base.orig/Tests/base/NSTimeZone/TestInfo
+++ gnustep-base/Tests/base/NSTimeZone/TestInfo
@@ -1,3 +1 @@
 # On unix systems force testing using our inbuilt time zone data.
-export GNUSTEP_BUILTIN_TZ=1
-
